app/models/notification.rb

```diff
@@ -0,0 +1,15 @@
+class Notification < ApplicationRecord
+  has_rich_text :description
+  validates :title, :description, presence: true
+  has_many :notify_users, dependent: :destroy
+  has_many :users, through: :notify_users
+  after_create :create_notify_user
+
+  private
+  def create_notify_user
+    @users = User.all
+    @users.each do |user|
+      NotifyUser.create(user_id: user.id, notification_id: id)
+    end
```

Comment:
We can do in one line also.
```suggestion
  self.users = User.all
```
Also This will create all records. 
Let me know if anything is wrong.